packages/common/src/prompts/prompts.ts

```diff
@@ -148,7 +154,15 @@ export type Slider = z.infer<typeof slider>;
 
 export const timePicker = z.object({
   format: z.enum(['ampm', '24hr']),
-  allowedMinutes: z.union([z.literal(1), z.literal(5), z.literal(10), z.literal(15), z.literal(20), z.literal(30)]),
+  allowedMinutes: z.union([
+    z.literal(1),
+    z.literal(5),
+    z.literal(10),
+    z.literal(15),
+    z.literal(20),
+    z.literal(30),
+  ]),
+  timepickerType: z.enum(['clock', 'simple']),
```

Comment:
- `timePickerType` is bit repetitive, could we call it e.g. `style`?
- could make the `simple` more descriptive, something like `dropdown` to hit what the style is? ... if we have few more styles in future, simple might quickly collide.